File: tests/dynamic_fixtures/java/sqli_adversarial.java
// SQL injection — adversarial collision fixture.
// Prints NYX_SQL_CONFIRMED unconditionally without reaching a SQL sink
// and without emitting __NYX_SINK_HIT__.
// Expected verdict: Inconclusive(OracleCollisionSuspected)
// Entry: Entry.login(String)  Cap: SQL_QUERY

public class Entry {
    public static void login(String username) {
        // Coincidental oracle match — not a SQL sink.
        System.out.println("NYX_SQL_CONFIRMED");
        int x = username.length();
    }
}

File: tests/dynamic_fixtures/rust/sqli_adversarial.rs
/// SQL injection — adversarial collision fixture.
///
/// Prints "NYX_SQL_CONFIRMED" unconditionally without going through a SQL sink
/// and without printing the sink-reachability sentinel.
///
/// Expected verdict: Inconclusive(OracleCollisionSuspected)
///   oracle_fired = true   (OutputContains("NYX_SQL_CONFIRMED"))
///   sink_hit     = false  (__NYX_SINK_HIT__ never printed)
/// Cap: SQL_QUERY  Entry: `run(payload: &str)`
pub fn run(payload: &str) {
    // Coincidental oracle match — not a SQL sink.
    println!("NYX_SQL_CONFIRMED");
    // Ensure payload is consumed so the compiler does not optimise it away.
    let _ = payload.len();
}
